modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/AgentLauncher.java

```diff
@@ -172,10 +181,18 @@
 
         IOException ioCause = X.cause(e, IOException.class);
 
-        if (ioCause != null && "404".equals(ioCause.getMessage())) {
-            log.error("You are using outdated version of Web agent. Please download latest version of Web agent from Web console");
+        if (ioCause != null) {
+            if ("404".equals(ioCause.getMessage())) {
+                log.error("You are using outdated version of Web agent. Please download latest version of Web agent from Web console.");
```

Comment:
This error also can happens when you are trying to connect to different application instead of web console

Reply:
Message changed.